AFTests/AlgoStore/AlgoStoreTestsInstanceNotRequired.cs

```diff
@@ -305,5 +300,48 @@ public partial class AlgoStoreTestsInstanceNotRequired : AlgoStoreTestDataFixtur
                 }
             });
         }
+
+        [Test, Description("AL-578")]
+        [Category("AlgoStore")]
+        [TestCase("Lykke.AlgoStore.CSharp.Algo.Implemention.ExecutableClass.Test")]
+        [TestCase("Lykke.AlgoStore.CSharp.Algo.Implemention.ExecutableClassTest")]
+        [TestCase("Lykke.AlgoStore.CSharp.Algo.Implemention.Test")]
+        [TestCase("Lykke.AlgoStore.CSharp.Algo")]
+        [TestCase("Lykke.AlgoStore.CSharp.Algo.ExecutableClass")]
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase(" ")]
+        [TestCase("     ")]
```

Comment:
I am embarrassed, what is that?)

Reply:
Different iterations of the same test case using different param for each test